### ISSUE TITLE
Prevent invalid sessions on restart of application

### ISSUE THREAD

**Issue by dj-hedgehog** [koajs/koa-redis#31]
I am using koa-generic-session with koa-redis. Everytime I restart my application all my sessions are invalid. I use it together with Passport and need to login again. How can I prevent this?

**Comment by DaAwesomeP** [comment]
You can use [`session.client`](https://github.com/koajs/koa-redis#sessionclient) to directly access the Redis library functions and clear the table.

**Comment by dj-hedgehog** [comment]
How will that help me keeping the data after a restart? I want my sessions to remain valid.

**Comment by DaAwesomeP** [comment]
I'm sorry, I misunderstood. Are you explicitly setting keys?

**Comment by dj-hedgehog** [comment]
Yes, I do but right now I only set them like this: `app.keys = ['keys', 'keykeys'];`

**Comment by DaAwesomeP** [comment]
The only thing that I can think of is that the TTLs are running out or that Redis is clearing the DB after restart. Have you setup persistence in Redis?
